tests/unit/test_helper.py

```diff
@@ -64,21 +64,6 @@ def test_np_int():
     assert hash2bytes(np.int64(a)) == hash2bytes(a)
 
 
-def test_hash():
```

Comment:
have these functions been removed? I have not seen the removal of these functions in this PR right?

Reply:
Haha, so predictable. When I removed this test, I knew you would write that.

`new_doc_id` was renamed to `get_content_hash` and `new_doc_hash` was removed. So there is no point in testing forward-backward converting. I thought about a test like
```
hash_ = get_content_hash(d)
assert hash_ == hash2id(id2hash(hash_))
```
But this already shows, that the semantic of `id` and `hash` are not precise anymore (Beware, it is NOT `id2hash(hash2id(hash_))`!). I will have another thought about this tomorrow with a fresh mind.